common/djangoapps/dark_lang/tests.py

```diff
@@ -42,33 +43,70 @@ def setUp(self):
             enabled=True
         ).save()
 
-    def process_request(self, language_session_key=UNSET, accept=UNSET, preview_lang=UNSET, clear_lang=UNSET):
+    def process_post(self, language_session_key=UNSET, accept=UNSET, preview_lang=UNSET, clear_lang=UNSET):
```

Comment:
Nit: Rename to make it clear that this method builds and makes a request to the dark-lang endpoint.
Maybe something like `process_darklang_request`.